# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-system-opus.diff
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-133.0.6943.60/third_party/webrtc/modules/audio_coding/codecs/opus/opus_inst.h.vanilla	2025-02-12 19:11:28.346495674 +0100
+++ chromium-133.0.6943.60/third_party/webrtc/modules/audio_coding/codecs/opus/opus_inst.h	2025-02-12 19:12:35.871629176 +0100
@@ -16,8 +16,8 @@
 #include "rtc_base/ignore_wundef.h"
 
 RTC_PUSH_IGNORING_WUNDEF()
-#include "third_party/opus/src/include/opus.h"
-#include "third_party/opus/src/include/opus_multistream.h"
+#include <opus/opus.h>
+#include <opus/opus_multistream.h>
 RTC_POP_IGNORING_WUNDEF()
 
 struct WebRtcOpusEncInst {
